mne/io/ctf/tests/test_ctf.py

```diff
@@ -302,4 +302,162 @@ def test_saving_picked(tmpdir, comp_grade):
+       139130, 141390, 143660, 145748, 147889, 150205, 152528, 154646,  # noqa
+       156897, 159191, 161446, 163722, 166077, 168467, 170624, 172519,  # noqa
+       174719, 176886, 179062, 181405, 183709, 186034, 188454, 190330,  # noqa
+       192660, 194682, 196834, 199161, 201035, 203008, 204999, 207409,  # noqa
+       209661, 211895, 213957, 216005, 218040, 220178, 222137, 224305,  # noqa
+       226297, 228654, 230755, 232909, 235205, 237373, 239723, 241762,  # noqa
+       243748, 245762, 247801, 250055, 251886, 254252, 256441, 258354,  # noqa
+       260680, 263026, 265048, 267073, 269235, 271556, 273927, 276197,  # noqa
+       278436, 280536, 282691, 284933, 287061, 288936, 290941, 293183,  # noqa
+       295369, 297729, 299626, 301546, 303449, 305548, 307882, 310124,  # noqa
+       312374, 314509, 316815, 318789, 320981, 322879, 324878, 326959,  # noqa
+       329341, 331200, 331201, 333469, 335584, 337984, 340143, 342034,  # noqa
+       344360, 346309, 348544, 350970, 353052, 355227, 357449, 359603,  # noqa
+       361725, 363676, 365735, 367799, 369777, 371904, 373856, 376204,  # noqa
+       378391, 380800, 382859, 385161, 387093, 389434, 391624, 393785,  # noqa
+       396093, 398214, 400198, 402166, 404104, 406047, 408372, 410686,  # noqa
+       413029, 414975, 416850, 418797, 420824, 422959, 425026, 427215,  # noqa
+       429278, 431668
+    ]) - 1  # Fieldtrip has 1 sample difference with MNE
+    ENVENT_NOT_PRESENT_IN_STIMCH_IDX = [44, 152]
+
+    from mne.datasets.brainstorm import bst_raw
+    from mne.io.ctf.markers import Markers
+    from mne.io.ctf.res4 import _read_res4
+    from mne import find_events
+    import itertools
+
+    data_path = bst_raw.data_path()
+    raw_path = (data_path + '/MEG/bst_raw/' +
+                'subj001_somatosensory_20111109_01_AUX-f.ds')
+
+    mm = Markers(raw_path)
+    res4 = _read_res4(raw_path)
+
+    latencies = []
+    labels = []
+    for current_marker_type in mm.keys():
+        labels.append([current_marker_type] * len(mm[current_marker_type]))
+        for trialnum, synctime in mm[current_marker_type]:
+            latencies.append(
+                res4['pre_trig_pts'] +
+                (trialnum * res4['nsamp']) +
+                round(synctime * res4['sfreq'])
+            )
+
+    unshuffling = np.argsort(latencies)
+
+    latencies = np.array(latencies)[unshuffling]
+    _labels = list(itertools.chain.from_iterable(labels))
+    labels = [_labels[x] for x in unshuffling]
+
+    assert_array_equal(latencies, EXPECTED_LATENCIES)
+
+    events_in_raw_sti = find_events(read_raw_ctf(raw_path, preload=True))[:, 0]
+    latencies_ = np.delete(latencies, ENVENT_NOT_PRESENT_IN_STIMCH_IDX)
+
+    assert_array_equal(  # For whatever reason 44, 151 and 152 do not match
+        np.delete(latencies_, [44, 151, 152]),
+        np.delete(events_in_raw_sti, [44, 151, 152])
+    )
+
+
+def test_fieldtrip_stuff():
```

Comment:
This should be removed before merging. This is here just as doc in case we have to trace back what happened in this PR.